src/System.CommandLine.Tests/Binding/ModelBindingCommandHandlerTests.cs

```diff
@@ -443,14 +479,82 @@ public void Apply(InvocationContext context)
                 o => o.Value.Should().Be("123")),
 
             BindingTestCase.Create<FileInfo>(
-                Path.Combine(Directory.GetCurrentDirectory(), "file1.txt"),
-                o => o.FullName.Should().Be(Path.Combine(Directory.GetCurrentDirectory(), "file1.txt"))),
+                Path.Combine(ExistingDirectory(), "file1.txt"),
+                o => o.FullName
+                      .Should()
+                      .Be(Path.Combine(ExistingDirectory(), "file1.txt"))),
 
             BindingTestCase.Create<FileInfo[]>(
-                $"{Path.Combine(Directory.GetCurrentDirectory(), "file1.txt")} {Path.Combine(Directory.GetCurrentDirectory(), "file2.txt")}",
+                $"{Path.Combine(ExistingDirectory(), "file1.txt")} {Path.Combine(ExistingDirectory(), "file2.txt")}",
                 o => o.Select(f => f.FullName)
                       .Should()
-                      .BeEquivalentTo(new[] { Path.Combine(Directory.GetCurrentDirectory(), "file1.txt"), Path.Combine(Directory.GetCurrentDirectory(), "file2.txt") })),
+                      .BeEquivalentTo(new[]
+                      {
+                          Path.Combine(ExistingDirectory(), "file1.txt"),
```

Comment:
There seems to be an (arguably accurate) assumption that all calls to `ExistingDirectory()` return the same value. I am wondering if it would be better to cache the value in a local and use that to make it more obvious when we are expecting values to match.